app/iot/devices.py

```diff
@@ -1,64 +1,80 @@
-import time
+import asyncio
```

Comment:
Please note: attach the screenshot of your script results (the console) to the PR.